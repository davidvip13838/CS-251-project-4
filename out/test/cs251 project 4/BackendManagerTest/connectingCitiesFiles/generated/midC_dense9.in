500
25 43
42 47
21 49
1 15
23 39
20 36
19 27
2 16
5 31
39 25
23 41
11 45
30 13
49 44
34 49
23 16
37 36
12 28
14 50
41 29
4 5
47 7
8 45
35 36
45 6
3 23
11 37
23 49
27 12
26 21
6 24
37 11
45 30
46 12
45 5
0 42
37 46
40 36
0 18
50 23
7 47
2 30
15 13
48 16
39 26
32 32
45 4
10 4
10 7
34 41
4 21
5 10
36 30
50 42
48 44
0 30
24 6
31 21
46 33
4 25
9 23
3 40
38 6
5 38
47 41
37 47
1 18
10 11
3 42
17 40
15 6
34 47
2 23
33 24
20 33
30 9
37 30
22 17
26 37
36 8
6 1
2 29
1 24
13 3
28 4
1 50
8 41
49 50
19 21
18 47
46 16
0 50
42 50
38 46
39 45
17 30
48 45
29 42
45 47
49 41
37 43
40 10
5 44
19 39
34 27
11 21
34 24
21 25
44 45
22 7
39 37
30 12
17 21
7 0
37 0
38 17
42 46
16 18
9 5
6 19
29 17
29 1
21 18
17 39
8 19
22 47
28 18
34 5
15 9
13 37
12 15
0 14
47 39
44 27
38 8
7 42
10 27
34 38
36 18
44 47
46 23
22 32
17 20
29 30
4 18
15 33
29 49
3 2
21 29
48 38
3 25
16 44
41 2
9 6
13 47
41 9
27 17
9 0
35 43
16 13
45 20
28 31
46 37
9 28
35 30
42 6
33 43
27 13
13 13
26 18
29 5
36 11
18 48
39 7
13 5
18 19
24 7
20 46
2 36
12 38
16 48
49 20
16 30
21 2
36 10
43 19
27 25
23 36
16 7
44 49
23 33
17 0
25 6
12 48
6 17
19 25
32 12
29 25
27 46
43 30
39 9
10 21
13 2
47 48
48 41
15 23
10 31
50 32
22 31
8 26
31 6
19 4
45 17
26 35
31 22
19 3
33 25
16 34
17 37
12 17
30 0
31 2
42 16
47 32
18 21
35 9
28 50
47 40
2 26
32 42
43 38
35 41
29 20
10 41
28 22
34 21
11 48
14 24
8 50
37 41
44 32
14 47
4 48
20 23
41 46
25 44
37 42
36 19
21 33
48 19
1 20
16 10
32 17
24 0
17 35
7 2
38 12
27 37
31 28
24 49
23 47
14 45
11 29
15 40
23 44
5 12
8 13
47 2
18 17
34 44
0 5
21 34
48 2
28 0
0 33
17 5
5 17
11 15
11 35
22 48
37 6
9 41
14 15
11 32
16 42
13 27
47 21
11 2
45 38
42 27
11 6
48 33
27 32
25 5
44 2
5 41
30 49
35 35
42 43
0 28
31 32
6 5
32 0
12 23
14 0
8 22
39 4
42 34
12 19
11 20
39 32
5 49
12 36
41 27
13 16
20 26
2 5
8 43
29 31
27 29
17 44
9 4
42 21
47 12
13 21
43 4
1 17
8 32
4 28
17 19
43 29
50 18
29 19
18 4
11 42
44 46
15 31
29 6
17 32
34 30
4 12
43 7
50 43
43 24
36 15
17 22
40 3
5 50
10 33
8 40
47 28
25 0
2 48
12 40
18 6
22 11
46 24
43 50
27 2
49 48
23 37
2 35
6 26
27 38
16 22
25 22
2 41
49 22
26 33
14 28
4 1
39 29
8 31
34 1
29 41
5 24
26 22
33 40
2 39
1 8
22 45
36 9
2 11
32 20
42 36
31 37
24 26
11 23
14 44
28 38
40 31
11 10
11 14
17 25
27 40
4 39
16 17
1 42
6 31
21 22
16 4
0 1
2 14
0 17
16 41
39 14
7 3
28 36
15 7
34 31
30 10
2 22
46 45
21 44
44 15
13 48
47 37
40 15
20 32
13 31
29 23
7 18
19 42
17 41
40 4
22 18
29 18
19 2
26 10
42 23
18 35
18 44
47 20
46 44
7 48
37 5
29 28
26 24
24 21
9 36
24 28
38 49
30 21
48 8
4 49
22 0
34 25
21 17
25 8
36 41
19 13
9 24
30 47
25 34
42 40
36 33
40 41
15 20
16 12
9 17
31 45
35 19
15 8
47 47
8 15
27 26
27 39
43 45
20 21
10 45
5 26
8 27
21 12
7 8
21 37
18 16
15 45
33 18
9 46
42 28
9 30
13 44
34 45
5 11
19 20
24 48
49 36
44 38
17 11
3 36
1 22
35 31
20 43
37 26
50 26
17 1
24 1
18 28
0 40
30 46
