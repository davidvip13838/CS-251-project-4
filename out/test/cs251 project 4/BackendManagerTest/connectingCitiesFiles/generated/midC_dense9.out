793.2553144369627